### PR TITLE
Explicitly import $ in modules where it's used

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

**Comment by brianhelba** [comment]
@girder/developers PTAL, ready for review.

File: importjq.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+clients/web/src/views/widgets/GroupMembersWidget.js
+clients/web/src/views/widgets/GroupModsWidget.js
+clients/web/src/views/widgets/HierarchyWidget.js
+clients/web/src/views/widgets/ItemBreadcrumbWidget.js
+clients/web/src/views/widgets/ItemListWidget.js
+clients/web/src/views/widgets/MetadataWidget.js
+clients/web/src/views/widgets/SearchFieldWidget.js
+clients/web/src/views/widgets/UploadWidget.js
+plugins/curation/web_client/views/HierarchyWidget.js
+plugins/item_tasks/web_client/main.js
```
Comment (zachmullen) [inline importjq.txt]: What's this file?

Comment (brianhelba) [inline importjq.txt]: My mistake.

**Comment by brianhelba** [comment]
@zachmullen Fixed.

**Pull request merged**